openshift/templates/nodejs.json

```diff
@@ -4,17 +4,23 @@
   "metadata": {
     "name": "nodejs-example",
     "annotations": {
-      "description": "A simple Node.js application with no database",
-      "iconClass": "icon-nodejs",
-      "tags": "instant-app,nodejs"
+      "description": "An example Node.js application with no database",
+      "tags": "instant-app,nodejs",
+      "iconClass": "icon-nodejs"
```

Comment:
@mfojtik we do: https://github.com/openshift/origin/blob/master/assets/app/styles/_openshift-logos-icon.less#L64